pkg/scheduler/scheduler.go

```diff
@@ -727,6 +715,18 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 			return
 		}
 
+		// Bind volumes first before Pod
```

Comment:
comment need updated.

Reply:
Thanks @wgliang ^_^   I didn't change the order between `bindVolumes` and `bindPod`. I just mv `bindVolumes` after `permit` plugin because I find it is useless to bind Volumes if the `permit` rejects the pod.